docs.sh

```diff
@@ -0,0 +1,11 @@
+set -xe
+
+export LC_CTYPE=C.UTF-8
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+echo generating docs
+# I assume this is bad
+nix build .#haddock
+echo serving docs
+python3 -m http.server --directory ./result/share/doc 8080
```

Comment:
`./result` only contains the result of your _last_ `nix build` (which may be a library build, not a haddock build, for example), which happens to be correct here, but there must be a more robust way of doing this.

Reply:
you can pass `${self.packages.${system}.haddock}` to the script and serve that

Reply:
Self doesn't seem to have a packages attribute. It seems like I would need to get the output of the flake using mlabs-tooling ie. psm but I don't know how to do that.